spring-cloud-gateway-server/src/test/java/org/springframework/cloud/gateway/handler/predicate/GatewayPredicateVisitorTests.java

```diff
@@ -16,25 +16,40 @@
 
 package org.springframework.cloud.gateway.handler.predicate;
 
-import java.util.ArrayList;
```

Comment:
Changing the order of imports will cause a checkstyle failure

Reply:
This is my IDEA's automatic code formatting, and I will turn it off